### PR TITLE
Feature/share collections

### PR DESCRIPTION
**Pull request by JJGothard**
Hi @SueSmith , this is for a release that's already in production in the app, fyi. Thanks for the review! 👍

### CODE REVIEW

File: src/pages/docs/postman/collections/sharing-collections.md [hunk 1]
```diff
@@ -62,21 +62,27 @@ The **SHARE COLLECTION** modal appears. It offers three ways to share a collecti
 
 [![share sidebar](https://assets.postman.com/postman-docs/githubusercontent2.png)](https://assets.postman.com/postman-docs/githubusercontent2.png)
 
-## Sharing collections in another workspace
+## Sharing collections with another workspace
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/sharing-collections.md:65]: Need to update the toc whenever a heading changes because it breaks the links

File: src/pages/docs/postman/collections/sharing-collections.md [hunk 2]
```diff
@@ -62,21 +62,27 @@ The **SHARE COLLECTION** modal appears. It offers three ways to share a collecti
...
 
 Select this option to share a collection to another workspace or with a workspace member. If shared to a team workspace, the collection will be visible to others in the team.
 
-You can set team default permissions to either view-only or edit; only after sharing a collection. First, you share the collection and then assign a role. You also will be able to grant existing team members individual user permissions. Remember, the **Manage Roles** modal will not be available until the collection is shared.
+After you have shared the collection, you can set your default permissions. You can assign a view-only or edit-only permission to the whole team, or you can grant individual user permissions to each team member. The **Manage Roles** modal will automatically appear at the end of the collection sharing process.
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/sharing-collections.md:69]: thinking "after sharing a collection" maybe reads a bit clearer? or even "you can set default permissions on a shared collection" for people who haven't read the previous bit..

File: src/pages/docs/postman/collections/sharing-collections.md [hunk 3]
```diff
@@ -62,21 +62,27 @@ The **SHARE COLLECTION** modal appears. It offers three ways to share a collecti
...
 
-[![share collections](https://assets.postman.com/postman-docs/WS-share-collection-dashboard.png)](https://assets.postman.com/postman-docs/WS-share-collection-dashboard.png)
+If you want to share the collection but keep it in the current workspace as well, click **Share collection**. If you want the collection to be removed from the current workspace after you share it, click **Share collection and remove from this workspace**. The default selection is **Share collection**.
 
-> If you remove a collection from a workspace, it will no longer be available in that workspace. To restore it to the workspace, share it again. If the collection is deleted and you need to restore access to it, you will need to [recover it](/docs/postman/collections/managing-collections/#recover-a-collection) first.
+> When you remove a collection from a workspace, all mocks, montitors, and integrations associated with it will also be removed.
+
+![Adjust roles](https://assets.postman.com/postman-docs/Adjust+roles+with+shared+collections.jpg)
+
+Adjust your roles and permissions, and click **Save Roles**.
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/sharing-collections.md:83]: would maybe reword to indicate they don't _need_ to adjust them, "select roles for collaborators in the workspace" or something

File: src/pages/docs/postman/collections/sharing-collections.md [hunk 4]
```diff
@@ -62,21 +62,27 @@ The **SHARE COLLECTION** modal appears. It offers three ways to share a collecti
...
+
+> To restore a collection to a workspace, share it again. If the collection is deleted and you need to restore access to it, you will need to [recover it](/docs/postman/collections/managing-collections/#recover-a-collection) first.
```
Comment (SueSmith) [inline src/pages/docs/postman/collections/sharing-collections.md:85]: could we add a bit of detail to indicate what we mean by restore? I think the removed sentence from previous version sets some context that's missing now..

File: src/pages/docs/postman/collections/sharing-collections.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/collections/sharing-collections.md:65]: Updated.

File: src/pages/docs/postman/collections/sharing-collections.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman/collections/sharing-collections.md:69]: @SueSmith , do you mean like "You can set default permissions on a shared collection" as the first sentence? And leave the rest of the paragraph as is?

File: src/pages/docs/postman/collections/sharing-collections.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman/collections/sharing-collections.md]: Done :)

File: src/pages/docs/postman/collections/sharing-collections.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman/collections/sharing-collections.md]: Done!

File: src/pages/docs/postman/collections/sharing-collections.md (hunk 2 above)
Comment (SueSmith) [reply src/pages/docs/postman/collections/sharing-collections.md:69]: that would work yep 👍

Comment (JJGothard) [reply src/pages/docs/postman/collections/sharing-collections.md]: Done

**Comment by JJGothard** [comment]
@SueSmith , all of the changes have been made, thanks!
